.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+name: Release Binary
+
+on:
+  push:
+    tags:
+      - v*
+
+permissions:
+  contents: write
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      -
+        name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
```

Comment:
Please bump up the go version in [this file](https://github.com/openebs/openebsctl/blob/master/.github/workflows/go.yml). It's good to have a single go version for all CI.

Reply:
Bumped to 1.16